### ISSUE TITLE
[High] JAWS+ Chrome: Error message is not announced by the screen reader. 

### ISSUE THREAD

**Issue by SumitDiyora** [Expensify/App#9557]
## Action Performed:
1. Using JAWS+ Chrome, open URL staging.new.expensify.com
2. Select any chat from the page.
3. Click on the Call button and the Request a call page will appear.
5. Leave the form fields blank and click on the "Call me" button, a error message will appear as a nudge.
6. Observe if there is any screen reader announcement.

## Expected Result:
The error message should be announced by the screen reader and the nudge should have controls to dismiss it according to user requirement. 

## Actual Result:
The error message appears as a nudge and disappears after a few seconds. Also, the error message is not announced by the screenreader.

## Workaround:
Yes

## Area issue was found in:
Request a Call screen

## Failed WCAG checkpoints
3.3.1 and 2.2.1

## User impact:
Screen reader users and cognitive users will not be aware of the error messages.

## Suggested resolution:
Providing role="alert" for error messages will make the screen reader announce the error messages. As a best practice, make the error messages available as text beside their respective fields if not then provide controls to dismiss the error messages according to the user's consent. 

## Platform:
- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:*v1.1.78-7*
**Reproducible in staging?:*Yes*
**Reproducible in production?:*Yes*
**Issue reported by:*Sumit*

![error message not announced by scrren reader](https://user-images.githubusercontent.com/98817464/175508731-7ac567a4-9200-4b4d-ae33-89e4dcef6a8f.jpg)
